src/app/cli/src/cli_parser.rs

```diff
@@ -1184,6 +1184,15 @@ pub fn cli() -> Command {
                                     .value_name("FMT")
                                     .value_parser(["shell", "json", "yaml"]),
                             ]),
+                        Command::new("diagnose")
+                            .about("Run basic system diagnose check")
+                            .args([
+                                Arg::new("output-format")
+                                    .long("output-format")
+                                    .short('o')
+                                    .value_name("FMT")
+                                    .value_parser(["shell", "json", "yaml"]),
+                            ]),
```

Comment:
This switch is not passed into the command, and is not influencing the output report anyhow.

Reply:
Forgot to remove it after last changes